src/Compilers/Core/Portable/WellKnownTypes.cs

```diff
@@ -675,7 +677,7 @@ public static bool IsWellKnownType(this WellKnownType typeId)
         public static bool IsValueTupleType(this WellKnownType typeId)
```

Comment:
>IsValueTupleType [](start = 27, length = 16)

Would you please put a comment on this that it is used for testing only?

Reply:
This method is also used in product code (`CSharpCompilation.GetWellKnownType`)